pallets/delegation/src/mock.rs

```diff
@@ -380,17 +468,28 @@ impl ExtBuilder {
 		self
 	}
 
+	// TODO: Remove optional input ext
 	pub fn build(self, ext: Option<sp_io::TestExternalities>) -> sp_io::TestExternalities {
 		let mut ext = if let Some(ext) = ext {
```

Comment:
For now, I kept this such that I did not need to alter tests which I did not need to touch. This will be fixed by #261. But if you want, I can refactor all tests in this PR as well.